dags/analyzer_helper/tests/integration/test_discord_extract_raw_info.py

```diff
@@ -0,0 +1,127 @@
+        result = extractor.extract(datetime(2023, 6, 30), recompute=True)
+
+        expected_result = sample_data
+
+        self.assertEqual(result, expected_result)
+
+    def test_extract_recompute_false(self):
+        rawmember_data = [
+            {
+                "_id": ObjectId("649fc4dfb65f6981303e32ef"),
+                "author_id": "DUMMY_DISCORD_ID_1",
+                "date": datetime(2023, 6, 30, 20, 28, 3, 494000),
+                "source_id": "DUMMY_MESSAGE_ID_1",
+                "metadata": {
+                    "channel_id": "DUMMY_CHANNEL_ID_1",
+                    "channel_name": "💬・general-chat",
+                    "thread_id": None,
+                    "thread_name": None,
+                },
+                "actions": [
+                    {
+                        "name": "message",
+                        "type": "emitter",
+                    }
+                ],
+                "interactions": [
+                    {
+                        "name": "mention",
+                        "users_engaged_id": [
+                            "DUMMY_DISCORD_ID_2",
+                            "DUMMY_DISCORD_ID_3",
+                        ],
+                        "type": "emitter",
+                    },
+                    {
+                        "name": "reaction",
+                        "users_engaged_id": ["44444444444444444", "91919191919191919"],
+                        "type": "receiver",
+                    },
+                ],
+            }
+        ]
+        self.rawmemberactivities_collection = self.platform_db["rawmemberactivities"]
+        self.rawmemberactivities_collection.delete_many({})
+        self.rawmemberactivities_collection.insert_many(rawmember_data)
+
+        sample_data = [
+            {
+                "_id": ObjectId("649fc4dfb65f6981303e32f0"),
+                "type": 0,
+                "author": "DUMMY_DISCORD_ID_1",
+                "content": "sample message",
+                "user_mentions": ["DUMMY_DISCORD_ID_2", "DUMMY_DISCORD_ID_3"],
+                "role_mentions": ["DUMMY_ROLE_ID_1", "DUMMY_ROLE_ID_2"],
+                "reactions": [
+                    "44444444444444444,thelounge",
+                    "91919191919191919,6373687382748239,👌",
+                ],
+                "createdDate": datetime(2023, 6, 30, 20, 28, 3, 494000),
+                "messageId": "DUMMY_MESSAGE_ID_1",
+                "channelId": "DUMMY_CHANNEL_ID_1",
+                "channelName": "💬・general-chat",
+                "threadId": None,
+                "threadName": None,
+                "isGeneratedByWebhook": False,
+            }
+        ]
+        self.rawinfos_collection.insert_many(sample_data)
+
+        extractor = DiscordExtractRawInfos(self.guild_id, self.platform_id)
+        result = extractor.extract(datetime(2023, 6, 30), recompute=False)
+
+        expected_result = sample_data
+
+        self.assertEqual(result, expected_result)
```

Comment:
We expect the result to be an empty array because we are already having the processed version of the raw data within `rawmemberactivities`.

I think the greater equality condition in `DiscordExtractRawInfos` for getting the rawinfos data is causing this. If I'm right, please consider updating the greater equality operator to greater operator

Link to the code: https://github.com/TogetherCrew/airflow-dags/blob/feat/analyzer-helpers-discord/dags/analyzer_helper/discord/discord_extract_raw_infos.py#L83C17-L83C51

Another option is to update the `$gte` to `$gt` in MongoDB query on line 86